addons/README.md

```diff
@@ -65,3 +65,58 @@ When using the graphical interface `cppcheck-gui`, the selection and configurati
 
 ![Screenshot](https://raw.githubusercontent.com/danmar/cppcheck/main/addons/doc/img/cppcheck-gui-addons.png)
 
+### Using namingng.py
```

Comment:
Just feedback on the struct and not he contents.

I guess this could be restructured into a "description", "configuration" and "usage" sub-section.

The invocation should be moved into a separate generic section as it applies to all addons.

Reply:
Good point, will change.